src/ring_theory/ideal/local_ring.lean

```diff
@@ -359,6 +359,18 @@ begin
   exact map_nonunit f a ha
 end
 
+/-- Applying `map` to the identity ring homomorphism gives the identity ring homomorphism. -/
+def map_id :
+  local_ring.residue_field.map(ring_hom.id R) = ring_hom.id (local_ring.residue_field R) :=
+ideal.quotient.ring_hom_ext $ ring_hom.ext $ λx, rfl
+
+/-- The composite of two `map`s is the `map` of the composite. -/
+lemma map_comp (f : T →+* R) (g : R →+* S)
+[is_local_ring_hom f] [is_local_ring_hom g] :
```

Comment:
```suggestion
lemma map_comp (f : T →+* R) (g : R →+* S) [is_local_ring_hom f] [is_local_ring_hom g] :
```
Hopefully this is less than 100 chars. Alternatively you should indent the second line.